packages/components/src/docs/Migration/ui-video.mdx

```diff
@@ -0,0 +1,87 @@
+# Migration Guide: From `ui-video` to `sd-video`
+
+The new `sd-video` is designed to replace the `ui-video`. This transition means that while `ui-video` functioned as an independent player, the `sd-video` component now serves as a wrapper, enabling users to provide video experiences with greater versatility. Instead of mainly providing content via attributes, the `sd-video` component uses slots to allow for more flexibility and customization.
+
+## 💾 Slots
+
+### ✨ New Slots
+
+#### [default]
+
+The default slot is used to pass a video player element.
+
+#### [play-icon]
+
+The slot for the video's play icon.
+
+#### [poster]
+
+Slot to specify an image to be shown before initial play of the wrapped video. This acts like the `poster` attribute on the native video tag.
+
+<hr />
+
+## ⚙️ Attributes
+
+### ✨ New Attributes
+
+#### [playing]
+
+Setting this to `true` will hide the play icon and the overlay.
+
+#### [isBelowBreakpoint]
```

Comment:
Is this a public attribute?

Reply:
Yup!
![image](https://github.com/solid-design-system/solid/assets/63849626/a9fd0c1c-6565-41ea-9ae5-b81fdc7c53fc)